src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -98,31 +100,33 @@ func majorVersion(version string) string {
 // copyNewest is a one-time function for using older config files in the 2.x series.
 // from 2.0.2 forward, Lantern will consider all major versions to be compatible and
 // will name them accordingly, as in "lantern-2.yaml".
-func copyNewest(file string) {
+func copyNewest(file string, existsFunc func(file string) (string, bool)) string {
```

Comment:
So, the main purpose of passing this function as a parameter is testability, am I correct?